pymilvus/__init__.py

```diff
@@ -47,7 +47,7 @@
     mkts_from_hybridts, mkts_from_unixtime, mkts_from_datetime,
     hybridts_to_unixtime, hybridts_to_datetime,
     # bulk_load, get_bulk_load_state,
-    reset_password, create_credential, update_credential, delete_credential, list_cred_users,
```

Comment:
Please change  `list_usernames` to `list_users` to keep consists with other functions